package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "eslint-plugin-ssr-friendly",
+  "name": "@cultureamp/eslint-plugin-ssr-friendly",
```

Comment:
you shouldn't rename package when it is fork. We shouldn't need to publish it to registery and just install from git
```suggestion
  "name": "eslint-plugin-ssr-friendly",
```

Reply:
I have put it back now